### ISSUE TITLE
[Constellation Vignette] This expression has type 'void' and can't be used

### ISSUE THREAD

**Issue by DigitalSolomon** [gskinnerTeam/flutter_vignettes#16]
In the constellation vignette, I'm getting this compilation error:

```
.../lib/widgets/nested_navigator.dart:23:29: Error: This expression has type 'void' and can't be used.
          return !navigator.pop();
```

This is running on Flutter (Channel stable, v1.17.4, on Mac OS X 10.14.5 18F132, locale en-US).

**Comment by Black-water-side** [comment]
me too!

**Comment by esDotDev** [comment]
Fixed with #25
